### PR TITLE
Add User-List

### PR DESCRIPTION
**Pull request by wasanderes94**
For #13  

- Put Composable Function `SearchBar` to seperate file for multi-use
- Created String Resources for UsersView
- Added `UserRow` for displaying `name` property, `(admin)` if `isAdmin` property is true and `balance` of user
- Added `UserList` for displaying multiple `UserRow`, separated with divider

### CODE REVIEW

File: app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt [hunk 1]
```diff
@@ -1,20 +1,96 @@
...
+
+@Composable
+fun UserRow(user: User) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Column {
```
Comment (Morphclue) [inline app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt:74]: You can remove this Column without breaking anything.

File: app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt [hunk 2]
```diff
@@ -1,20 +1,96 @@
...
+            // Not sure if there's a better inline solution
+            Text(
+                user.name + " ${if (user.isAdmin) "(${stringResource(id = R.string.user_admin)})" else ""}",
+                fontWeight = FontWeight.Medium
+            )
+        }
+        Column() {
```
Comment (Morphclue) [inline app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt:81]: You can also remove this Column.

**Review by Morphclue — CHANGES_REQUESTED** [review]
Otherwise the PR looks fine!

File: app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt [hunk 3]
```diff
@@ -1,20 +1,90 @@
...
+
+@Composable
+fun UserRow(user: User) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        // Not sure if there's a better inline solution
```
Comment (Morphclue) [inline app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt:73]: You can also remove this comment, but i will already give you my approve.

**Review by Morphclue — APPROVED** [review]

File: app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt (hunk 3 above)
Comment (wasanderes94) [reply app/src/main/java/org/feature/fox/coffee_counter/ui/UsersView.kt]: Removed it

**Pull request merged**